client/src/components/ProductInfo.jsx

```diff
@@ -0,0 +1,65 @@
+const React = require("react");
+import DescBullet from "./DescBullet.jsx";
+
+const addComma = function(num) {
+  return String(num).replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
impressive use of regex, not sure what for tho, perhaps comment when you're doing stuff like this